contrib/tlp/patches/00-Makefile.patch

```diff
@@ -0,0 +1,123 @@
+--- Makefile
++++ Makefile
+@@ -141,44 +141,51 @@
+ 
+ install-tlp: all
```

Comment:
you should probably submit these changes upstream because rewriting these patches is a pain